service/service_construction.go

```diff
@@ -111,6 +111,8 @@ func (s ConstructionService) ConstructionMetadata(
 			if input.Metadata != nil {
 				if input.Metadata.UnwrapBridgeTx {
 					gasLimit, err = s.getBridgeUnwrapTransferGasLimit(ctx, input.From, input.Value, input.Currency)
+				} else {
+					return nil, wrapError(errInvalidInput, "UnwrapBridgeTx must be populated if input.Metadata is provided")
```

Comment:
This would've caused the estimation to be 0 if there was metadata but no `UnwrapBridgeTx` boolean